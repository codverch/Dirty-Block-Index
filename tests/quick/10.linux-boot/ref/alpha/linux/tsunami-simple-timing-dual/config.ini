[root]
type=Root
children=system
checkpoint=
clock=2000000000
max_tick=0
output_file=cout
progress_interval=0

[debug]
break_cycles=

[exetrace]
intel_format=false
pc_symbol=true
print_cpseq=false
print_cycle=true
print_data=true
print_effaddr=true
print_fetchseq=false
print_iregs=false
print_opclass=true
print_thread=true
speculative=true
trace_system=client

[serialize]
count=10
cycle=0
dir=cpt.%012d
period=0

[stats]
descriptions=true
dump_cycle=0
dump_period=0
dump_reset=false
ignore_events=
mysql_db=
mysql_host=
mysql_password=
mysql_user=
project_name=test
simulation_name=test
simulation_sample=0
text_compat=true
text_file=m5stats.txt

[system]
type=LinuxAlphaSystem
children=bridge cpu0 cpu1 disk0 disk2 intrctrl iobus membus physmem sim_console simple_disk tsunami
boot_cpu_frequency=1
boot_osflags=root=/dev/hda1 console=ttyS0
console=/dist/m5/system/binaries/console
init_param=0
kernel=/dist/m5/system/binaries/vmlinux
mem_mode=timing
pal=/dist/m5/system/binaries/ts_osfpal
physmem=system.physmem
readfile=tests/halt.sh
system_rev=1024
system_type=34

[system.bridge]
type=Bridge
delay=0
queue_size_a=16
queue_size_b=16
write_ack=false

[system.cpu0]
type=TimingSimpleCPU
children=dtb itb
clock=1
cpu_id=-1
defer_registration=false
dtb=system.cpu0.dtb
function_trace=false
function_trace_start=0
itb=system.cpu0.itb
max_insts_all_threads=0
max_insts_any_thread=0
max_loads_all_threads=0
max_loads_any_thread=0
mem=system.physmem
profile=0
system=system

[system.cpu0.dtb]
type=AlphaDTB
size=64

[system.cpu0.itb]
type=AlphaITB
size=48

[system.cpu1]
type=TimingSimpleCPU
children=dtb itb
clock=1
cpu_id=-1
defer_registration=false
dtb=system.cpu1.dtb
function_trace=false
function_trace_start=0
itb=system.cpu1.itb
max_insts_all_threads=0
max_insts_any_thread=0
max_loads_all_threads=0
max_loads_any_thread=0
mem=system.physmem
profile=0
system=system

[system.cpu1.dtb]
type=AlphaDTB
size=64

[system.cpu1.itb]
type=AlphaITB
size=48

[system.disk0]
type=IdeDisk
children=image
delay=2000
driveID=master
image=system.disk0.image

[system.disk0.image]
type=CowDiskImage
children=child
child=system.disk0.image.child
read_only=false
table_size=65536

[system.disk0.image.child]
type=RawDiskImage
image_file=/dist/m5/system/disks/linux-latest.img
read_only=true

[system.disk2]
type=IdeDisk
children=image
delay=2000
driveID=master
image=system.disk2.image

[system.disk2.image]
type=CowDiskImage
children=child
child=system.disk2.image.child
read_only=false
table_size=65536

[system.disk2.image.child]
type=RawDiskImage
image_file=/dist/m5/system/disks/linux-bigswap2.img
read_only=true

[system.intrctrl]
type=IntrControl
cpu=system.cpu0

[system.iobus]
type=Bus
bus_id=0

[system.membus]
type=Bus
bus_id=1

[system.physmem]
type=PhysicalMemory
file=
latency=1
range=0:134217727

[system.sim_console]
type=SimConsole
children=listener
append_name=true
intr_control=system.intrctrl
listener=system.sim_console.listener
number=0
output=console

[system.sim_console.listener]
type=ConsoleListener
port=3456

[system.simple_disk]
type=SimpleDisk
children=disk
disk=system.simple_disk.disk
system=system

[system.simple_disk.disk]
type=RawDiskImage
image_file=/dist/m5/system/disks/linux-latest.img
read_only=true

[system.tsunami]
type=Tsunami
children=cchip console etherint ethernet fake_OROM fake_ata0 fake_ata1 fake_pnp_addr fake_pnp_read0 fake_pnp_read1 fake_pnp_read2 fake_pnp_read3 fake_pnp_read4 fake_pnp_read5 fake_pnp_read6 fake_pnp_read7 fake_pnp_write fake_ppc fake_sm_chip fake_uart1 fake_uart2 fake_uart3 fake_uart4 fb ide io pchip pciconfig uart
intrctrl=system.intrctrl
system=system

[system.tsunami.cchip]
type=TsunamiCChip
pio_addr=8803072344064
pio_latency=2
platform=system.tsunami
system=system
tsunami=system.tsunami

[system.tsunami.console]
type=AlphaConsole
cpu=system.cpu0
disk=system.simple_disk
pio_addr=8804682956800
pio_latency=2
platform=system.tsunami
sim_console=system.sim_console
system=system

[system.tsunami.etherint]
type=NSGigEInt
device=system.tsunami.ethernet
peer=Null

[system.tsunami.ethernet]
type=NSGigE
children=configdata
clock=0
config_latency=40
configdata=system.tsunami.ethernet.configdata
dma_data_free=false
dma_desc_free=false
dma_no_allocate=true
dma_read_delay=0
dma_read_factor=0
dma_write_delay=0
dma_write_factor=0
hardware_address=00:90:00:00:00:01
intr_delay=20000
pci_bus=0
pci_dev=1
pci_func=0
pio_latency=2
platform=system.tsunami
rss=false
rx_delay=2000
rx_fifo_size=524288
rx_filter=true
rx_thread=false
system=system
tx_delay=2000
tx_fifo_size=524288
tx_thread=false

[system.tsunami.ethernet.configdata]
type=PciConfigData
BAR0=1
BAR0Size=256
BAR1=0
BAR1Size=4096
BAR2=0
BAR2Size=0
BAR3=0
BAR3Size=0
BAR4=0
BAR4Size=0
BAR5=0
BAR5Size=0
BIST=0
CacheLineSize=0
CardbusCIS=0
ClassCode=2
Command=0
DeviceID=34
ExpansionROM=0
HeaderType=0
InterruptLine=30
InterruptPin=1
LatencyTimer=0
MaximumLatency=52
MinimumGrant=176
ProgIF=0
Revision=0
Status=656
SubClassCode=0
SubsystemID=0
SubsystemVendorID=0
VendorID=4107

[system.tsunami.fake_OROM]
type=IsaFake
pio_addr=8796093677568
pio_latency=2
pio_size=393216
platform=system.tsunami
system=system

[system.tsunami.fake_ata0]
type=IsaFake
pio_addr=8804615848432
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_ata1]
type=IsaFake
pio_addr=8804615848304
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_addr]
type=IsaFake
pio_addr=8804615848569
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read0]
type=IsaFake
pio_addr=8804615848451
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read1]
type=IsaFake
pio_addr=8804615848515
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read2]
type=IsaFake
pio_addr=8804615848579
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read3]
type=IsaFake
pio_addr=8804615848643
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read4]
type=IsaFake
pio_addr=8804615848707
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read5]
type=IsaFake
pio_addr=8804615848771
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read6]
type=IsaFake
pio_addr=8804615848835
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read7]
type=IsaFake
pio_addr=8804615848899
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_write]
type=IsaFake
pio_addr=8804615850617
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_ppc]
type=IsaFake
pio_addr=8804615848892
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_sm_chip]
type=IsaFake
pio_addr=8804615848816
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_uart1]
type=IsaFake
pio_addr=8804615848696
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_uart2]
type=IsaFake
pio_addr=8804615848936
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_uart3]
type=IsaFake
pio_addr=8804615848680
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fake_uart4]
type=IsaFake
pio_addr=8804615848944
pio_latency=2
pio_size=8
platform=system.tsunami
system=system

[system.tsunami.fb]
type=BadDevice
devicename=FrameBuffer
pio_addr=8804615848912
pio_latency=2
platform=system.tsunami
system=system

[system.tsunami.ide]
type=IdeController
children=configdata
config_latency=40
configdata=system.tsunami.ide.configdata
disks=system.disk0 system.disk2
pci_bus=0
pci_dev=0
pci_func=0
pio_latency=2
platform=system.tsunami
system=system

[system.tsunami.ide.configdata]
type=PciConfigData
BAR0=1
BAR0Size=8
BAR1=1
BAR1Size=4
BAR2=1
BAR2Size=8
BAR3=1
BAR3Size=4
BAR4=1
BAR4Size=16
BAR5=1
BAR5Size=0
BIST=0
CacheLineSize=0
CardbusCIS=0
ClassCode=1
Command=0
DeviceID=28945
ExpansionROM=0
HeaderType=0
InterruptLine=31
InterruptPin=1
LatencyTimer=0
MaximumLatency=0
MinimumGrant=0
ProgIF=133
Revision=0
Status=640
SubClassCode=1
SubsystemID=0
SubsystemVendorID=0
VendorID=32902

[system.tsunami.io]
type=TsunamiIO
frequency=1953125
pio_addr=8804615847936
pio_latency=2
platform=system.tsunami
system=system
time=1136073600
tsunami=system.tsunami

[system.tsunami.pchip]
type=TsunamiPChip
pio_addr=8802535473152
pio_latency=2
platform=system.tsunami
system=system
tsunami=system.tsunami

[system.tsunami.pciconfig]
type=PciConfigAll
bus=0
pio_latency=1
platform=system.tsunami
size=16777216
system=system

[system.tsunami.uart]
type=Uart8250
pio_addr=8804615848952
pio_latency=2
platform=system.tsunami
sim_console=system.sim_console
system=system

[trace]
bufsize=0
dump_on_exit=false
file=cout
flags=
ignore=
start=0

