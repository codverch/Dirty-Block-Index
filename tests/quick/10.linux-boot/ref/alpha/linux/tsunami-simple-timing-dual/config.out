[root]
type=Root
dummy=0

[system.physmem]
type=PhysicalMemory
file=
range=[0,134217727]
latency=1
zero=false

[system]
type=LinuxAlphaSystem
boot_cpu_frequency=500
physmem=system.physmem
mem_mode=timing
kernel=/dist/m5/system/binaries/vmlinux
console=/dist/m5/system/binaries/console
pal=/dist/m5/system/binaries/ts_osfpal
boot_osflags=root=/dev/hda1 console=ttyS0
readfile=tests/halt.sh
symbolfile=
init_param=0
system_type=34
system_rev=1024

[system.membus]
type=Bus
bus_id=1
clock=1000
width=64
responder_set=false
block_size=64

[system.intrctrl]
type=IntrControl
sys=system

[system.tsunami]
type=Tsunami
system=system
intrctrl=system.intrctrl

[system.membus.responder]
type=IsaFake
pio_addr=0
pio_latency=1
pio_size=8
ret_bad_addr=true
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.l2c]
type=BaseCache
size=4194304
assoc=8
block_size=64
latency=10000
mshrs=92
tgts_per_mshr=16
write_buffers=8
prioritizeRequests=false
protocol=null
trace_addr=0
hash_delay=1
repl=null
compressed_bus=false
store_compressed=false
adaptive_compression=false
compression_latency=0
block_size=64
max_miss_count=0
addr_range=[0,18446744073709551615]
split=false
split_size=0
lifo=false
two_queue=false
prefetch_miss=false
prefetch_access=false
prefetcher_size=100
prefetch_past_page=false
prefetch_serial_squash=false
prefetch_latency=100000
prefetch_degree=1
prefetch_policy=none
prefetch_cache_check_push=true
prefetch_use_cpu_id=true
prefetch_data_accesses_only=false

[system.bridge]
type=Bridge
req_size_a=16
req_size_b=16
resp_size_a=16
resp_size_b=16
delay=50000
nack_delay=4000
write_ack=false
fix_partial_write_a=false
fix_partial_write_b=true

[system.disk0.image.child]
type=RawDiskImage
image_file=/dist/m5/system/disks/linux-latest.img
read_only=true

[system.disk0.image]
type=CowDiskImage
child=system.disk0.image.child
image_file=
table_size=65536
read_only=false

[system.disk0]
type=IdeDisk
image=system.disk0.image
driveID=master
delay=1000000

[system.disk2.image.child]
type=RawDiskImage
image_file=/dist/m5/system/disks/linux-bigswap2.img
read_only=true

[system.disk2.image]
type=CowDiskImage
child=system.disk2.image.child
image_file=
table_size=65536
read_only=false

[system.disk2]
type=IdeDisk
image=system.disk2.image
driveID=master
delay=1000000

[system.cpu0.itb]
type=AlphaITB
size=48

[system.cpu0.dtb]
type=AlphaDTB
size=64

[system.cpu0]
type=TimingSimpleCPU
max_insts_any_thread=0
max_insts_all_threads=0
max_loads_any_thread=0
max_loads_all_threads=0
progress_interval=0
system=system
cpu_id=0
itb=system.cpu0.itb
dtb=system.cpu0.dtb
profile=0
do_quiesce=true
do_checkpoint_insts=true
do_statistics_insts=true
clock=500
phase=0
defer_registration=false
// width not specified
function_trace=false
function_trace_start=0
// simulate_stalls not specified

[system.cpu0.icache.protocol]
type=CoherenceProtocol
protocol=moesi
do_upgrades=true

[system.cpu0.icache]
type=BaseCache
size=32768
assoc=1
block_size=64
latency=1000
mshrs=4
tgts_per_mshr=8
write_buffers=8
prioritizeRequests=false
protocol=system.cpu0.icache.protocol
trace_addr=0
hash_delay=1
repl=null
compressed_bus=false
store_compressed=false
adaptive_compression=false
compression_latency=0
block_size=64
max_miss_count=0
addr_range=[0,18446744073709551615]
split=false
split_size=0
lifo=false
two_queue=false
prefetch_miss=false
prefetch_access=false
prefetcher_size=100
prefetch_past_page=false
prefetch_serial_squash=false
prefetch_latency=10000
prefetch_degree=1
prefetch_policy=none
prefetch_cache_check_push=true
prefetch_use_cpu_id=true
prefetch_data_accesses_only=false

[system.cpu0.dcache.protocol]
type=CoherenceProtocol
protocol=moesi
do_upgrades=true

[system.cpu0.dcache]
type=BaseCache
size=32768
assoc=4
block_size=64
latency=1000
mshrs=4
tgts_per_mshr=8
write_buffers=8
prioritizeRequests=false
protocol=system.cpu0.dcache.protocol
trace_addr=0
hash_delay=1
repl=null
compressed_bus=false
store_compressed=false
adaptive_compression=false
compression_latency=0
block_size=64
max_miss_count=0
addr_range=[0,18446744073709551615]
split=false
split_size=0
lifo=false
two_queue=false
prefetch_miss=false
prefetch_access=false
prefetcher_size=100
prefetch_past_page=false
prefetch_serial_squash=false
prefetch_latency=10000
prefetch_degree=1
prefetch_policy=none
prefetch_cache_check_push=true
prefetch_use_cpu_id=true
prefetch_data_accesses_only=false

[system.cpu1.itb]
type=AlphaITB
size=48

[system.cpu1.dtb]
type=AlphaDTB
size=64

[system.cpu1]
type=TimingSimpleCPU
max_insts_any_thread=0
max_insts_all_threads=0
max_loads_any_thread=0
max_loads_all_threads=0
progress_interval=0
system=system
cpu_id=1
itb=system.cpu1.itb
dtb=system.cpu1.dtb
profile=0
do_quiesce=true
do_checkpoint_insts=true
do_statistics_insts=true
clock=500
phase=0
defer_registration=false
// width not specified
function_trace=false
function_trace_start=0
// simulate_stalls not specified

[system.cpu1.icache.protocol]
type=CoherenceProtocol
protocol=moesi
do_upgrades=true

[system.cpu1.icache]
type=BaseCache
size=32768
assoc=1
block_size=64
latency=1000
mshrs=4
tgts_per_mshr=8
write_buffers=8
prioritizeRequests=false
protocol=system.cpu1.icache.protocol
trace_addr=0
hash_delay=1
repl=null
compressed_bus=false
store_compressed=false
adaptive_compression=false
compression_latency=0
block_size=64
max_miss_count=0
addr_range=[0,18446744073709551615]
split=false
split_size=0
lifo=false
two_queue=false
prefetch_miss=false
prefetch_access=false
prefetcher_size=100
prefetch_past_page=false
prefetch_serial_squash=false
prefetch_latency=10000
prefetch_degree=1
prefetch_policy=none
prefetch_cache_check_push=true
prefetch_use_cpu_id=true
prefetch_data_accesses_only=false

[system.cpu1.dcache.protocol]
type=CoherenceProtocol
protocol=moesi
do_upgrades=true

[system.cpu1.dcache]
type=BaseCache
size=32768
assoc=4
block_size=64
latency=1000
mshrs=4
tgts_per_mshr=8
write_buffers=8
prioritizeRequests=false
protocol=system.cpu1.dcache.protocol
trace_addr=0
hash_delay=1
repl=null
compressed_bus=false
store_compressed=false
adaptive_compression=false
compression_latency=0
block_size=64
max_miss_count=0
addr_range=[0,18446744073709551615]
split=false
split_size=0
lifo=false
two_queue=false
prefetch_miss=false
prefetch_access=false
prefetcher_size=100
prefetch_past_page=false
prefetch_serial_squash=false
prefetch_latency=10000
prefetch_degree=1
prefetch_policy=none
prefetch_cache_check_push=true
prefetch_use_cpu_id=true
prefetch_data_accesses_only=false

[system.simple_disk.disk]
type=RawDiskImage
image_file=/dist/m5/system/disks/linux-latest.img
read_only=true

[system.simple_disk]
type=SimpleDisk
system=system
disk=system.simple_disk.disk

[system.tsunami.fake_uart1]
type=IsaFake
pio_addr=8804615848696
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fake_uart2]
type=IsaFake
pio_addr=8804615848936
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fake_uart3]
type=IsaFake
pio_addr=8804615848680
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fake_uart4]
type=IsaFake
pio_addr=8804615848944
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fake_ppc]
type=IsaFake
pio_addr=8804615848891
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.cchip]
type=TsunamiCChip
pio_addr=8803072344064
pio_latency=1000
platform=system.tsunami
system=system
tsunami=system.tsunami

[system.tsunami.io]
type=TsunamiIO
pio_addr=8804615847936
pio_latency=1000
frequency=976562500
platform=system.tsunami
system=system
time=2009 1 1 0 0 0 3 1
year_is_bcd=false
tsunami=system.tsunami

[]
type=PciConfigAll
pio_latency=1
bus=0
size=16777216
platform=system.tsunami
system=system

[system.sim_console]
type=SimConsole
intr_control=system.intrctrl
output=console
port=3456
append_name=true
number=0

[system.tsunami.console]
type=AlphaConsole
sim_console=system.sim_console
disk=system.simple_disk
pio_addr=8804682956800
system=system
cpu=system.cpu0
platform=system.tsunami
pio_latency=1000

[system.tsunami.fake_ata1]
type=IsaFake
pio_addr=8804615848304
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fake_ata0]
type=IsaFake
pio_addr=8804615848432
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.pchip]
type=TsunamiPChip
pio_addr=8802535473152
pio_latency=1000
platform=system.tsunami
system=system
tsunami=system.tsunami

[system.tsunami.fake_pnp_read3]
type=IsaFake
pio_addr=8804615848643
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read2]
type=IsaFake
pio_addr=8804615848579
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read1]
type=IsaFake
pio_addr=8804615848515
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read0]
type=IsaFake
pio_addr=8804615848451
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read7]
type=IsaFake
pio_addr=8804615848899
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read6]
type=IsaFake
pio_addr=8804615848835
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read5]
type=IsaFake
pio_addr=8804615848771
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_read4]
type=IsaFake
pio_addr=8804615848707
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_write]
type=IsaFake
pio_addr=8804615850617
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fb]
type=BadDevice
devicename=FrameBuffer
pio_addr=8804615848912
system=system
platform=system.tsunami
pio_latency=1000

[system.tsunami.ethernet.configdata]
type=PciConfigData
VendorID=4107
DeviceID=34
Command=0
Status=656
Revision=0
ProgIF=0
SubClassCode=0
ClassCode=2
CacheLineSize=0
LatencyTimer=0
HeaderType=0
BIST=0
BAR0=1
BAR1=0
BAR2=0
BAR3=0
BAR4=0
BAR5=0
CardbusCIS=0
SubsystemVendorID=0
SubsystemID=0
ExpansionROM=0
InterruptLine=30
InterruptPin=1
MinimumGrant=176
MaximumLatency=52
BAR0Size=256
BAR1Size=4096
BAR2Size=0
BAR3Size=0
BAR4Size=0
BAR5Size=0

[system.tsunami.ethernet]
type=NSGigE
system=system
platform=system.tsunami
min_backoff_delay=4000
max_backoff_delay=10000000
configdata=system.tsunami.ethernet.configdata
pci_bus=0
pci_dev=1
pci_func=0
pio_latency=1000
config_latency=20000
clock=0
dma_desc_free=false
dma_data_free=false
dma_read_delay=0
dma_write_delay=0
dma_read_factor=0
dma_write_factor=0
dma_no_allocate=true
intr_delay=10000000
rx_delay=1000000
tx_delay=1000000
rx_fifo_size=524288
tx_fifo_size=524288
rx_filter=true
hardware_address=00:90:00:00:00:01
rx_thread=false
tx_thread=false
rss=false

[system.tsunami.etherint]
type=NSGigEInt
peer=null
device=system.tsunami.ethernet

[system.tsunami.fake_OROM]
type=IsaFake
pio_addr=8796093677568
pio_latency=1000
pio_size=393216
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.uart]
type=Uart8250
pio_addr=8804615848952
pio_latency=1000
platform=system.tsunami
sim_console=system.sim_console
system=system

[system.tsunami.fake_sm_chip]
type=IsaFake
pio_addr=8804615848816
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.fake_pnp_addr]
type=IsaFake
pio_addr=8804615848569
pio_latency=1000
pio_size=8
ret_bad_addr=false
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

[system.tsunami.ide.configdata]
type=PciConfigData
VendorID=32902
DeviceID=28945
Command=0
Status=640
Revision=0
ProgIF=133
SubClassCode=1
ClassCode=1
CacheLineSize=0
LatencyTimer=0
HeaderType=0
BIST=0
BAR0=1
BAR1=1
BAR2=1
BAR3=1
BAR4=1
BAR5=1
CardbusCIS=0
SubsystemVendorID=0
SubsystemID=0
ExpansionROM=0
InterruptLine=31
InterruptPin=1
MinimumGrant=0
MaximumLatency=0
BAR0Size=8
BAR1Size=4
BAR2Size=8
BAR3Size=4
BAR4Size=16
BAR5Size=0

[system.tsunami.ide]
type=IdeController
system=system
platform=system.tsunami
min_backoff_delay=4000
max_backoff_delay=10000000
configdata=system.tsunami.ide.configdata
pci_bus=0
pci_dev=0
pci_func=0
pio_latency=1000
config_latency=20000
disks=system.disk0 system.disk2

[system.iobus]
type=Bus
bus_id=0
clock=1000
width=64
responder_set=true
block_size=64

[system.toL2Bus]
type=Bus
bus_id=0
clock=1000
width=64
responder_set=false
block_size=64

[system.toL2Bus.responder]
type=IsaFake
pio_addr=0
pio_latency=1
pio_size=8
ret_bad_addr=true
update_data=false
warn_access=
ret_data8=255
ret_data16=65535
ret_data32=4294967295
ret_data64=18446744073709551615
platform=system.tsunami
system=system

